README.md

```diff
@@ -5,20 +5,26 @@
 
 CIDAR: Culturally-Relevant Instruction Dataset For Arabic. CIDAR contains 10,000 instructions and their outputs. The dataset was created by selecting around 9.2k samples from [Alpagasus](https://huggingface.co/datasets/mlabonne/alpagasus) dataset then translating it using ChatGPT. In addition, we append that with around 800 instructions from the webiste [Ask the teacher](https://learning.aljazeera.net/ar/asktheteacher). All the 10,000 samples are reviewed by around 10 reviewers. 
 
-## CIDAR Annotation
-First install the requirements `pip install -r requirements.txt` then run the app using flask `flask --app app run `.
+## 1. CIDAR Annotation
+### To run the app:
+1. Install the requirements `pip install -r requirements.txt`.
+1. Run the app using flask `flask --app app run `.
```

Comment:
should be 1. and 2. ?

Reply:
Yes, you are right thanks.

Reply:
It seems to work both ways, but I will change it as it's easier to read.